tasks_downstream.py

```diff
@@ -108,7 +108,33 @@ def write_code_workspace_file(c, cw_path=None):
         pass  # Nevermind, we start with a new config
     # Static settings
     cw_config.setdefault("settings", {})
-    cw_config["settings"].update({"search.followSymlinks": False})
+    cw_config["settings"].update(
+        {
+            "editor.formatOnSave": True,
```

Comment:
Beware. Enabling this in the `odoo` folder will break your workflow when pushing a PR to odoo.

Also, we don't want to `black` OCA modules in versions prior to 13.0.

Please check those 2 points and report back.

Reply:
Yes, I am working on that :smiley: Thanks!